src/ui/ui_idle_menu_nbgl.c

```diff
@@ -48,8 +48,13 @@ void onQuitCallback(void) {
 
 static bool settingsNavCallback(uint8_t page, nbgl_pageContent_t* content) {
     if (page == 0) {
+        content->type = INFOS_LIST;
+        content->infosList.nbInfos = NB_INFO_FIELDS;
+        content->infosList.infoTypes = (const char**) infoTypes;
+        content->infosList.infoContents = (const char**) infoContents;
```

Comment:
```suggestion
        content->infosList.infoTypes = infoTypes;
        content->infosList.infoContents = infoContents;
```
down cast not necessary anymore.

Reply:
Done